README.md

```
@@ -1,36 +1,527 @@
+- [Overview](#-overview)
+- [Features](#-features)
+- [Tech Stack](#-tech-stack)
+- [Getting Started](#-getting-started)
+- [Project Structure](#-project-structure)
+- [Components](#-components)
+- [Styling](#-styling)
+- [SEO & Performance](#-seo--performance)
+- [Deployment](#-deployment)
+- [Browser Support](#-browser-support)
+- [Contributing](#-contributing)
+- [License](#-license)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)